cfg/validate_test.go

```diff
@@ -303,3 +303,122 @@ func Test_IsTtlInSecsValid_ValidScenarios(t *testing.T) {
 		})
 	}
 }
+
+func Test_isValidWriteConfig_ErrorScenarios(t *testing.T) {
+	var testCases = []struct {
+		testName    string
+		writeConfig WriteConfig
+	}{
+		{"zero_block_size", WriteConfig{
+			BlockSizeMb:                       0,
+			CreateEmptyFile:                   false,
+			ExperimentalEnableStreamingWrites: true,
+			GlobalMaxBlocks:                   -1,
+			MaxBlocksPerFile:                  -1,
+		}},
+		{"negative_block_size", WriteConfig{
+			BlockSizeMb:                       -1,
+			CreateEmptyFile:                   false,
+			ExperimentalEnableStreamingWrites: true,
+			GlobalMaxBlocks:                   -1,
+			MaxBlocksPerFile:                  -1,
+		}},
+		{"-2_global_max_blocks", WriteConfig{
+			BlockSizeMb:                       10,
+			CreateEmptyFile:                   false,
+			ExperimentalEnableStreamingWrites: true,
+			GlobalMaxBlocks:                   -2,
+			MaxBlocksPerFile:                  -1,
+		}},
+		{"0_global_max_blocks", WriteConfig{
+			BlockSizeMb:                       10,
+			CreateEmptyFile:                   false,
+			ExperimentalEnableStreamingWrites: true,
+			GlobalMaxBlocks:                   0,
+			MaxBlocksPerFile:                  -1,
+		}},
+		{"1_global_max_blocks", WriteConfig{
+			BlockSizeMb:                       10,
+			CreateEmptyFile:                   false,
+			ExperimentalEnableStreamingWrites: true,
+			GlobalMaxBlocks:                   1,
+			MaxBlocksPerFile:                  -1,
+		}},
+		{"-2_max_blocks_per_file", WriteConfig{
+			BlockSizeMb:                       10,
+			CreateEmptyFile:                   false,
+			ExperimentalEnableStreamingWrites: true,
+			GlobalMaxBlocks:                   20,
+			MaxBlocksPerFile:                  -2,
+		}},
+		{"0_max_blocks_per_file", WriteConfig{
+			BlockSizeMb:                       10,
+			CreateEmptyFile:                   false,
+			ExperimentalEnableStreamingWrites: true,
+			GlobalMaxBlocks:                   20,
+			MaxBlocksPerFile:                  0,
+		}},
+		{"1_max_blocks_per_file", WriteConfig{
+			BlockSizeMb:                       10,
+			CreateEmptyFile:                   false,
+			ExperimentalEnableStreamingWrites: true,
+			GlobalMaxBlocks:                   20,
+			MaxBlocksPerFile:                  1,
+		}},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.testName, func(t *testing.T) {
+			assert.Error(t, isValidWriteStreamingConfigs(&tc.writeConfig))
+		})
+	}
+}
+
+func Test_isValidWriteConfig_ValidScenarios(t *testing.T) {
```

Comment:
```suggestion
func Test_isValidWriteConfig_validScenarios(t *testing.T) {
```

Reply:
Could not find the style guide for this so renamed it as per the example given in [go tip](https://g3doc.corp.google.com/company/teams/go-community/gotip/episodes/50.md?cl=head).